### PR TITLE
Small hack to get the back button working

### PR DESCRIPTION
**Pull request by Lukewh**

### CODE REVIEW

**Comment by Lukewh** [comment]
**QA**
- Login
- Click the search box
- Click on a charm
- Click back
- Be back on store page

This needs to be removed when the whole 'store url is broken' things are fixed.

File: jujugui/static/gui/src/app/state/state.js [hunk 1]
```diff
@@ -499,7 +499,7 @@ const State = class State {
     // If we have root paths in the URL then we can ignore everything else.
     if (state.root) {
       // If there is anything after this then it's an invalid URL.
-      if (parts.length > 1) {
+      if (parts.length > 1 && parts[0] !== 'store') {
```
Comment (jaycee) [inline jujugui/static/gui/src/app/state/state.js]: Add an XXX about when this should be removed, please?

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6708/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6709/

**Comment by Lukewh** [comment]
Fixes https://github.com/juju/juju-gui/issues/2391 (for connectivity)

**Comment by jaycee** [comment]
QA OK.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6712/

**Comment by Lukewh** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**